src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -324,9 +318,33 @@ export default {
 			);
 			this.enableRelendingExp = version === 'b';
 		}
+
+		const recommendedArray = [
+			...cachedRecommendedLoans,
+			{ id: 0 }, { id: 0 },
+			{ id: 0 }, { id: 0 },
+			{ id: 0 }, { id: 0 },
+			{ id: 0 }, { id: 0 },
+		];
+
+		let relendingArray = [
+			...cachedRecommendedLoans,
+		];
+
+		/* eslint-disable max-len */
+		if (this.enableFiveDollarsNotes) {
+			if ((userBalance > 0 && userBalance < 10) || (userBalance > 20 && userBalance < 50)) relendingArray = relendingArray.slice(0, 2);
```

Comment:
Unfortunately we aren't guaranteed that these loans will all have at least $25 remaining unreserved. This means that in `LendCtaExp.vue` we also need to check/fallback to the unreserved amount, like you have in the `multipleAmount` amount method. With the logic in `multipleAmount`, there could be a button with a larger amount than what is unreserved for a loan (and wouldn't align with the number calculated in `multipleAmount`.